### PR TITLE
Add User skills section

### PR DESCRIPTION
**Pull request by Rheniery**

### CODE REVIEW

File: src/app/users/_presenters/components/UserForm/_presenters/Skills/_presenters/components/SkillForm/index.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+function SkillForm({
+  userSkill,
+  control,
+  skills,
+  remove,
+  index,
+}: Props): JSX.Element {
+  const { selectedSkill, setSelectedSkill } = useSkillFormController();
+
+  useEffect(() => {
```
Comment (kaiomagalhaes) [inline src/app/users/_presenters/components/UserForm/_presenters/Skills/_presenters/components/SkillForm/index.tsx:31]: this goes into the controller

File: src/app/users/_presenters/components/UserForm/_presenters/Skills/_presenters/data/utils.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import skillLevelKeys from '@/app/_domain/enums/skillLevelKeys';
+
+export const LEVEL_LIST = [
+  `${skillLevelKeys.Beginner}`,
```
Comment (kaiomagalhaes) [inline src/app/users/_presenters/components/UserForm/_presenters/Skills/_presenters/data/utils.ts:4]: I don't think you need the interpolation here, just add the key directly on the list

File: src/app/users/_presenters/components/UserForm/_presenters/Skills/index.tsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+const defaultUserSkill: UserSkill = {
+  lastAppliedYear: 2010,
+  level: skillLevelKeys.Beginner,
+  yearsOfExperience: 0,
+  skillId: undefined,
+  userId: undefined,
+};
+
+function Skills({ userId }: Props): JSX.Element {
+  const { skills, isLoading } = useSkillsController();
```
Comment (kaiomagalhaes) [inline src/app/users/_presenters/components/UserForm/_presenters/Skills/index.tsx:29]: this all should go into the controller

**Review by kaiomagalhaes — CHANGES_REQUESTED** [review]

File: src/app/users/_presenters/components/UserForm/_presenters/Skills/index.tsx (hunk 3 above)
Comment (Rheniery) [reply src/app/users/_presenters/components/UserForm/_presenters/Skills/index.tsx:29]: Hey @kaiomagalhaes, I understand that every state we should send to the Controller but this file is handled with a few controllers, should we encapsulate all of them into a new one?

**Pull request merged**